### ISSUE TITLE
The two impact indicators are a bit confusing

### ISSUE THREAD

**Issue by anxolin** [cowprotocol/cowswap#1627]

**Comment by elena-zh** [comment]
Based on the discussion agreed on: 
 -  Not showing USD price impact percentage on the Limit order page at all
 -  Show different warning message according to cases: 
1. Limit Price impact > USD price impact, all >5%: show limit price warning
![image](https://user-images.githubusercontent.com/70885163/214315059-03c43984-3db4-4026-8bbe-39a6927027fe.png)

2. Limit Price impact = USD price impact, all >5%: show limit price warning 
![image](https://user-images.githubusercontent.com/70885163/214315059-03c43984-3db4-4026-8bbe-39a6927027fe.png)

3. Limit Price impact < USD price impact, Limit Price impact > 5%:  ---> **TBD**

4. Limit Price impact < USD price impact, Limit Price impact <= 5%, USD price impact>5%  ---> show different warning like "You loose..." **TBD** 
![image](https://user-images.githubusercontent.com/70885163/214316714-31b8478e-2abe-4cfb-babe-54c7b4cf989d.png)

5. Limit Price impact and USD price impact <=5%: don't show warnings
6.  Limit Price impact >5% and NO USD price impact (yellow warning): ---> **TBD (show limit price warning?)**
![image](https://user-images.githubusercontent.com/70885163/214315059-03c43984-3db4-4026-8bbe-39a6927027fe.png)
7. Limit Price impact <=5% and NO USD price impact (yellow warning): ---> **TBD (show only 1 yellow warning?)**
![image](https://user-images.githubusercontent.com/70885163/214317879-992f851a-4695-4e17-adc3-8aee059b4725.png)

**Comment by anxolin** [comment]
3. Limit Price impact < USD price impact, Limit Price impact > 5%: ---> TBD

I think we said, we show a message with the "price impact warning" 

Something like: 

```
Loosing XXXX$. 
According on our estimations, the received tokes are worth XXXX$, but the selling tokens YYYYY$.
Please, double check your limit price and make sure you are comfortable with the current value.
```
cc/ @MindyCoW pls make your own counter-proposal on the warning.

**Comment by anxolin** [comment]
> 4. Limit Price impact < USD price impact, Limit Price impact <= 5%, USD price impact>5% ---> show different warning like "You loose..." TBD

For me this is the same case as 3.

**Comment by anxolin** [comment]
> 6. Limit Price impact >5% and NO USD price impact (yellow warning): ---> TBD (show limit price warning?)

Yes, show the limit price warning. It's more severe than the no USD impact. It already forces the user to check the limit price

**Comment by anxolin** [comment]
> 7 Limit Price impact <=5% and NO USD price impact (yellow warning): ---> TBD (show only 1 yellow warning?)

Correct!

**Comment by elena-zh** [comment]
> > 4. Limit Price impact < USD price impact, Limit Price impact <= 5%, USD price impact>5% ---> show different warning like "You loose..." TBD
> 
> For me this is the same case as 3.

Not the same case. When Limit Price impact <= 5 , there is no Limit price warning
![image](https://user-images.githubusercontent.com/70885163/214322578-efcf0eac-6c07-4b1f-a061-0a02e02d86cc.png)

**Comment by anxolin** [comment]
> Not the same case. When Limit Price impact <= 5 , there is no Limit price warning

Yes, what I mean, is that the outcome is the same as case 3. We just show https://github.com/cowprotocol/cowswap/issues/1627#issuecomment-1402042871

**Comment by MindyCoW** [comment]
Summarizing: 

1. Limit Price impact > USD price impact, all >5%: **show limit price warning**

2. Limit Price impact = USD price impact, all >5%: **show limit price warning**

3. Limit Price impact < USD price impact, Limit Price impact > 5%: **Losing $XXXX! 
Due to low liquidity, you will only receive an estimated $XXXX worth of [BUY TOKEN] for this trade. That's $XXXX less than the value of your [SELL TOKEN]. Please double check your limit price and be sure you are comfortable with the current value.**

4. Limit Price impact < USD price impact, Limit Price impact <= 5%, USD price impact>5%: **Same as above**

5. Limit Price impact and USD price impact <=5%: **don't show warnings**

6. Limit Price impact >5% and NO USD price impact: **show limit price warning**

7. Limit Price impact <=5% and NO USD price impact: **show yellow warning**

Finally, the price impact percentage will be removed in all cases. Only the limit price percentage (above the limit price field) will remain.

**Comment by MindyCoW** [comment]
I want to add another point here: IMO, the limit price percentage should turn **red** when the limit price is below market price in all cases. Many users in these current tests are getting confused about whether they should set a limit price _higher_ or _lower_ than current market price. We need to make it more clear when their current price is _below_ market price, just so they spend less time trying to figure out if a higher price is good or bad.

**Comment by MindyCoW** [comment]
Is this planned to be implemented soon?

**Comment by anxolin** [comment]
I don't think so. We worked on the limit order reiteration (for more than people were happy with), and this was not added. 

I don't think is hard to do the change, just convince Anant to put it for Q2 if you feel is important
